create_blog_post_using_azure_openai_and_publish_in_medium/main.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/http;
+import ballerina/io;
+import ballerinax/azure_openai.text;
+import ballerinax/medium;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+configurable string mediumToken = ?;
+
+const API_VERSION = "2023-03-15-preview";
+
+public function main(string title) returns error? {
+    final medium:Client medium = check new ({auth: {token: mediumToken}});
+    final text:Client azureOpenAI = check new (
+        config = {httpVersion: http:HTTP_1_1, auth: {apiKey: openAIToken}},
```

Comment:
We might have to remove the http version based on the fix done to the connector.